web/wp-content/themes/dctx-v2/searchform.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-aie1bv.php, but found phpcs-scan-aIe1Bv.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).